### PR TITLE
add logic for fizzbuzz

### PR DESCRIPTION
**Pull request by vuong2-pdx**
Implement logic for fizzbuzz

### CODE REVIEW

File: hw1/01-fizzbuzz.js [hunk 1]
```diff
@@ -7,13 +7,25 @@ Write a program that writes all the numbers from 1 to 100, with some exceptions:
 
 Use console.log() to write the proper output to the command line.
 
-**/
+* */
```
Comment (vuong2-pdx) [inline hw1/01-fizzbuzz.js:10]: I was able to set up my eslint to use Airbnb's rules using links posted on Leshi and Ethan's PR - thanks Leshi and Ethan!

**Review by ethanSE — APPROVED** [review]
Here you are initializing variable "output" to an empty string. This value is never read. Other options include declaring the variable without assigning a value or initialize it with the value of i.

There's a slight difference between strings in single quotes and back ticks which are for template literals.

File: hw1/01-fizzbuzz.js [hunk 2]
```diff
@@ -7,13 +7,23 @@ Write a program that writes all the numbers from 1 to 100, with some exceptions:
...
 
 const fizzbuzz = () => {
-  // Add your code here
-};
+    for (let i = 1; i <= 100; i += 1) {
+        let output = i
+        if (i % 3 === 0 && i % 5 === 0) {
+            output = 'fizzbuzz'
```
Comment (vuong2-pdx) [inline hw1/01-fizzbuzz.js:16]: did a Replace All for backticks - forgot to replace them after experimenting with templates :o)

**Review by sashetov — APPROVED** [review]
I think the code works and is passing the linter as well as has gone through the corrections suggested by ethan earlier so its good to merge.

**Comment by JeremyIV** [comment]
Your solution doesn't quite match the behavior described in the homework assignment, because it prints the number each time, regardless of whether "fizz" or "buzz" was printed as well.

**Pull request merged**